tools/ktx/command_create.cpp

```diff
@@ -3,10 +3,11 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "command.h"
+#include "encode_utils_common.h"
 #include "platform_utils.h"
 #include "metrics_utils.h"
-#include "compress_utils.h"
-#include "encode_utils.h"
+#include "deflate_utils.h"
+#include "basis_utils.h"
```

Comment:
For consistency I think `encode_utils_basis` and `encode_utils_astc` (when we get to it) fit better with `encode_utils_common`.

I'd like to move the ASTC encode utils out of command_create either in this PR or a PR dedicated to that purpose. In other words, before a PR that adds the new encode_astc command or updates the encode command.